projects/demo/src/pages/documentation/angular/examples/1-nested/component.ts

```diff
@@ -0,0 +1,20 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {MaskitoPredicate} from '@maskito/angular';
+import {MaskitoOptions} from '@maskito/core';
+
+@Component({
+    selector: 'nested-doc-example-1',
+    templateUrl: './template.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NestedDocExample1 {
+    value = '';
+
+    readonly nameMask: MaskitoOptions = {
+        mask: /^[a-zA-Z\s]+$/,
+        postprocessor: ({value, selection}) => ({value: value.toUpperCase(), selection}),
+    };
+
+    readonly predicate: MaskitoPredicate = element =>
```

Comment:
Should we drop it for `Default` case ?

Reply:
I keep it here because that's what I show in TypeScript tab. I think it's ok since this tab is kind of for both examples.